ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__cloud-governance.yaml

```diff
@@ -20,6 +20,17 @@ tests:
     test:
     - ref: openshift-qe-cloud-governance-tagging
   timeout: 8h0m0s
+- as: perfscale-tagging
+  capabilities:
+  - intranet
+  cron: 0 */2 * * *
```

Comment:
Please change the timing of the cron, and spread the jobs across the clock please